addons/account/models/account_move.py

```diff
@@ -3904,6 +3907,14 @@ def write(self, vals):
                 if not self.env.user.has_group('account.group_account_manager'):
                     raise UserError(_('The Journal Entry sequence is not conform to the current format. Only the Accountant can change it.'))
                 move.journal_id.sequence_override_regex = False
+            if 'date' in vals:
```

Comment:
Remove code. Unnecessary, if we slightly change the behviour of the Audit button in the lock wizard. (based on account.move date, rather than track.values dates)